docs/catalog/cloud/scale-infrastructure-with-user-load.md

```diff
@@ -0,0 +1,42 @@
+---
+version: 1.0
+submitted_by: franziska-nttdata-hesselfeld
+published_date: TBD
+category: cloud
+tags: 
+ - compute
```

Comment:
please include `cloud` in the tag list as well.

Reply:
Done